Created Header and headline section

V-Blaze opened the pull request:
The HTML and CSS design for the header section of the Portfolio website.

V-Blaze committed:
installed npm dependencies and set-up linters

V-Blaze committed:
created index.html and style.css file

V-Blaze committed:
Added text and images to nav-bar and heading section

V-Blaze committed:
added styling to the nav-bar and the heading section

V-Blaze committed:
fixed stylelint errors from style.css

V-Blaze committed:
fixed more stylelint errors from style.css

V-Blaze committed:
fixed single-space and semi-column stylelint errors from style.css

christianbiring1 commented on index.html:
```diff
@@ -0,0 +1,42 @@
+<body>
+	<section class="hero-section">
+		<nav class="nav-bar">
+			<h1 class="logo">
+				My Logo
+			</h1>
```
Kindly make sure your logo is wrapped inside an anchor tag `<a>` and not  header`<h1>`

christianbiring1 commented on index.html:
```diff
@@ -0,0 +1,42 @@
+			<h3 class="connect">
+				LET’S CONNECT
+			</h3>
+			<div class="social-links">
+				<a href="https://twitter.com/blaze_valentine"><img src="./assets/images/twitter_icon.svg" alt="" srcset=""></a>
+				<a href="https://www.linkedin.com/in/valentine-blaze"><img src="./assets/images/Linkedin_icon.svg" alt="" srcset=""></a>
+				<a href="http://medium.com"><img src="./assets/images/medium_icon.svg" alt="" srcset=""></a>
+				<a href="https://github.com/V-Blaze"><img src="./assets/images/github_icon.svg" alt="" srcset=""></a>
+				<a href="http://"><img src="./assets/images/Vector.svg" alt="" srcset=""></a>
+			</div>
```
- [ ] Kindly consider wrapping all the social links inside a `ul`  and not a `div`. and each one of them inside a `li` for semantic purpose
- [ ] Kindly make sure you add the values for your alternative text `alt` attribute. As you can see their utility [here](https://www.w3schools.com/tags/att_area_alt.asp#:~:text=Definition%20and%20Usage,user%20uses%20a%20screen%20reader).

christianbiring1 commented on index.html:
```diff
@@ -0,0 +1,42 @@
+	
+	
+
+	</nav>
```
Kindly remove this `nav` tag here. as it's not suppose to be there

christianbiring1 commented on style.css:
```diff
@@ -0,0 +1,84 @@
+  color: #6070ff;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  height: 90%;
+  align-items: start;
+  justify-content: center;
+  background-image: url("./assets/images/header-shapes-mobile.svg");
+  background-size: 100%;
```
Kindly make sure the `margin` and `padding` are met with the one provided on the Figma design.
As you can see your design
![Screenshot from 2022-09-21 18-14-54](https://user-images.githubusercontent.com/97100091/191558211-04fcd7cf-b309-49ac-94c3-7cdd3b6d38f1.png)

comparing to the origin design
![Screenshot from 2022-09-21 18-21-48](https://user-images.githubusercontent.com/97100091/191558472-0fc7ee51-a2e4-45bb-8aec-26e5d98e7239.png)

christianbiring1 reviewed (changes requested):
Hi @V-Blaze , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![163947610-e4d849d5-e69c-49b9-90bd-a4191f4e2856](https://user-images.githubusercontent.com/97100091/191558701-8ed50201-3123-435c-89d8-29edcb66c64d.gif)


## Highlights
:heavy_check_mark: All linters are green
:heavy_check_mark: You have used `Flex` as required.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Document your project in a professional way
- [ ] Add a descriptive summary to your PR


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

V-Blaze committed:
wrapped social links in ul and li for better sematics

V-Blaze commented:
Below are the major changes in this new pull request

- Changed logo wrapper from 'Div tag' to 'a tag' for better semantics.
- Moved the social icons into 'ul' & 'li' tags.
- Made proper adjustments to the padding and margins of the home design according to the Figma design.

codecaiine reviewed (approved):
Hi @V-Blaze, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Descriptive PR :heavy_check_mark: 
- Great design :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
